.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,4 +5,4 @@
 ### To-Do
 
 [//]: # (Stay ahead of things, add list items here!)
-- [ ] Clean up commit history
+- [ ] Clean up commit history and use conventional commits
```

Comment:
I just closed #953 since it requires a 3rd-party orb, usage of which is not authorized (and will not be) on our circleci account. But I do think adding a lint step to our ci build which validates this is better than simply stating that commits should be "conventional" (tracked in #979). Either way, it's not a blocker for this pr and this is great for the time being. 

One thing I would change here is to just make this a hyperlink to "https://www.conventionalcommits.org/en/v1.0.0/" for easy reference